validation/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+authors = ["Valentin Tolmer <valentin@tolmer.fr>"]
+description = "Validation logic for LLDAP"
+edition = "2021"
+homepage = "https://github.com/lldap/lldap"
+license = "GPL-3.0-only"
+name = "lldap_validation"
+repository = "https://github.com/lldap/lldap"
+version = "0.6.0"
+
+[features]
+default = []
+js = []
+
+[dependencies.derive_more]
+features = ["debug", "display"]
+default-features = false
+version = "1"
+
+# For WASM targets, use the JS getrandom.
```

Comment:
Is that necessary? Because it's a dependency somewhere? Can we disable that dependency via features maybe?

Reply:
I'm assuming you mean the WASM stuff? Yeah, since it's also validating in the frontend. Any suggestions?

Reply:
This is only necessary if we depend on random, which we don't use here. Does it actually break if you remove it?

Reply:
They could all be removed. Done.